src/core/utils/helpers/general.ts

```diff
@@ -127,6 +127,14 @@ export const daysBetweenTodayAndDate = (date: string) => {
   // and we need it to be rounded up
   return Math.ceil(inputDate.diff(today, "day", true));
 };
+export const daysBetweenDates = (firstData: string, secondDate: string) => {
+  const inputFirstDate = dayjs(new Date(firstData));
+  const inputSecondDate = dayjs(new Date(secondDate));
+
+  // Math.ceil 0 diff last param true is because "diff()" rounds the number down
+  // and we need it to be rounded up
+  return Math.ceil(inputFirstDate.diff(inputSecondDate, "day", true));
```

Comment:
Why ceil here?
Imagine a situation where this function is deciding if a user has surpassed their 60 day loan-period and is being fined.
if say 60.09 days have passed and the number is ceiled, 61 days suddenly have passed and you are being fined, even tho only 60 full days have passed, therefore making the effective loan-period only 59.99 days instead of 60.99 days.
Make any sense?

Reply:
I've added a todo.